internal/services/gcp_reservation_service.go

```diff
@@ -2,10 +2,9 @@ package services
 
 import (
 	"errors"
-	"net/http"
-
 	"github.com/go-chi/render"
 	"github.com/lzap/dejq"
+	"net/http"
```

Comment:
linter dislikes this move, net/http needs to be above the github (external) imports.
This is difference between go fmt and go imports style.
In case you're using goland, you can change the style at [`Settings/Preferences | Editor | Code Style | <language> | Imports`](https://www.jetbrains.com/help/go/creating-and-optimizing-imports.html#optimize-imports) we are using the `goimports` sorting style :)

Reply:
@avitova what I do is simply running `make fmt` and it will run all formatters we require.